docs/DeveloperGuide.md

```diff
@@ -86,9 +86,10 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. TrackIt@NUS shows an error message.
 
       Use case resumes at step 2.
+      
 
 *{More to be added}*
```

Comment:
Should we remove this `*{More to be added}*` line?

Reply:
I am not sure; we will update this section constantly, but the line might be unnecessary. I'll remove it for now.

Reply:
Update: fixed.